pkg/istio-agent/sds-agent.go

```diff
@@ -405,6 +386,10 @@ func (sa *Agent) newWorkloadSecretCache() (workloadSecretCache *cache.SecretCach
 		}
 	}
 
+	if sa.secOpts.TokenExchangers == nil {
```

Comment:
Why was this moved ?

Reply:
I am fixing a bug that breaks Mesh CA. The plugin name was not set at earlier point. So token exchanger is empty.